certified-connectors/Industrial App Store/apiProperties.json

```diff
@@ -28,7 +28,9 @@
         }
       }
     },
-    "iconBrandColor": "#CCCCCC",
-    "capabilities": []
+    "iconBrandColor": "#ffffff",
```

Comment:
@gsmcmurtrie Please note that white background color for icon is not allowed.

Documentation - [Icon](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#icon)